server/shared/mail/templates/comment.mjml

```diff
@@ -30,5 +28,33 @@
         </mj-text>
       </mj-column>
     </mj-section>
+    <mj-section padding="0 10px">
+      <mj-column border-top="1px solid #ccc">
+        <mj-text
+          mj-class="content"
+          padding-left="0"
+          color="#444"
+          font-size="16px">
+          Open the activity by clicking the button below:
```

Comment:
Forgot to mention initially; please replace activity terminology with activity label (config).